src/libraries/Common/src/System/Security/Cryptography/RSACng.EncryptDecrypt.cs

```diff
@@ -198,7 +198,7 @@ private unsafe bool TryEncryptOrDecrypt(ReadOnlySpan<byte> data, Span<byte> dest
         // Now that the padding mode and information have been marshaled to their native counterparts, perform the encryption or decryption.
         private unsafe byte[] EncryptOrDecrypt(SafeNCryptKeyHandle key, ReadOnlySpan<byte> input, AsymmetricPaddingMode paddingMode, void* paddingInfo, bool encrypt)
         {
-            int estimatedSize = KeySize / 8;
+            int estimatedSize = GetMaxOutputSize();
```

Comment:
This had been doing `KeySize / 8` and is now doing `(KeySize + 7) >>> 3`.  I assume this is fine, as this is an estimate and ceiling is better than floor.  But just checking...

Reply:
Yeah. Straight "divide by 8" was technically wrong but worked in practice. The ceiling calculation is "right".